.gitpod.yml

```diff
@@ -0,0 +1,26 @@
+image:
+  file: .gitpod.Dockerfile
+
+ports:
+  - port: 3000
+    onOpen: open-preview
+tasks:
+  - init: >
+      rvm autolibs disable &&
+      rvm install 2.5.1 &&
+      gem install bundler -v 1.17.3 &&
+      cp config/database.example.yml config/database.yml &&
```

Comment:
These instructions for rails app, Interactive book is a jekyll app, not rails

Reply:
Please review if it is correct now. I tried to fix that.